src/django_bird/conf.py

```diff
@@ -1,111 +1,32 @@
 from __future__ import annotations
 
-import warnings
-from contextlib import suppress
 from dataclasses import dataclass
 from dataclasses import field
 from pathlib import Path
-from typing import Any
-from typing import final
 
-import django.template
 from django.conf import settings
 
-from django_bird import hookimpl
-
 from ._typing import override
 from .utils import unique_ordered
 
 DJANGO_BIRD_SETTINGS_NAME = "DJANGO_BIRD"
 
+DJANGO_BIRD_BUILTINS = "django_bird.templatetags.django_bird"
+DJANGO_BIRD_FINDER = "django_bird.staticfiles.BirdAssetFinder"
+
 
 @dataclass
 class AppSettings:
     COMPONENT_DIRS: list[Path | str] = field(default_factory=list)
-    ENABLE_AUTO_CONFIG: bool = True
     ENABLE_BIRD_ATTRS: bool = True
-    _configurator: AutoConfigurator = field(init=False)
-
-    def __post_init__(self):
-        self._configurator = AutoConfigurator(self)
 
     @override
     def __getattribute__(self, __name: str) -> object:
         user_settings = getattr(settings, DJANGO_BIRD_SETTINGS_NAME, {})
         return user_settings.get(__name, super().__getattribute__(__name))
 
-    def autoconfigure(self) -> None:
-        if not self.ENABLE_AUTO_CONFIG:
-            return
-
-        warnings.warn(
-            "Autoconfiguration of django-bird is deprecated and has been moved to the "
-            "django-bird-autoconf plugin. Please install the new plugin from PyPI in your "
-            "project if you wish to keep this behavior, as this will be removed from the "
-            "core library in a future version.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        self._configurator.autoconfigure()
-
     def get_component_directory_names(self):
         return unique_ordered([*self.COMPONENT_DIRS, "bird"])
 
 
-@hookimpl
```

Comment:
Lol, I was running autoconfiguration twice I guess? Once in `app.ready()` and then in this hook which is called there too.